source/_slider.slim

```diff
@@ -0,0 +1,61 @@
+div id="carousel-hero" class="carousel slide" data-ride="carousel"
+  / Indicators 
+  ol class="carousel-indicators hero-indicators"
+    li data-target="#carousel-hero" data-slide-to="0" class="active current-indicator"
+    li data-target="#carousel-hero" class="current-indicator" data-slide-to="1"
+    li data-target="#carousel-hero" class="current-indicator" data-slide-to="2"
+    li data-target="#carousel-hero" class="current-indicator" data-slide-to="3"
+  / slides
+  div class="carousel-inner" role="listbox"
+    div class="item active"
+        img class="slide" src="images/slide-1.JPG" alt="..."
+          div class="img-tint"
+        div class="carousel-caption hero-caption"
+          h3 class="hero-heading"
+            | terapia indywidualna
+          p class="hero-paragraph"
+            | Zapraszamy osoby, które chcą lepiej zrozumieć siebie.
+          a class="btn btn-lg blue-button" href="#" role="button"
+            | więcej
+
+    div class="item"
+      img class="slide" src="images/slide-2.jpg" alt="..."
+        div class="img-tint"
+      div class="carousel-caption hero-caption"
+        h3 class="hero-heading"
+          | terapia dla dzieci i młodzieży
+        p class="hero-paragraph"
+          |  Gwarantujemy okazję pokonywania własnych barier w bezpiecznej i serdecznej atmosferze
+        a class="btn btn-lg blue-button" href="#" role="button"
+          | więcej
+
+    div class="item"
+      img class="slide" src="images/slide-3.jpg" alt="..."
+        div class="img-tint"
+      div class="carousel-caption hero-caption"
+        h3 class="hero-heading"
+          | depresja
+        p class="hero-paragraph"
+          | Jeżeli czujesz że nie potrafisz sam poradzić sobie z rozwiązaniem konkretnego problemu, zapraszamy do skorzystania z porady u psychologa.
+        a class="btn btn-lg blue-button" href="#" role="button"
+          | więcej
+
+    div class="item"
+      img class="slide" src="images/slide-4.JPG" alt="..."
+        div class="img-tint"
+      div class="carousel-caption hero-caption"
+        h3 class="hero-heading"
+          | terapia grupowa
+        p class="hero-paragraph"
+          | Sesje grupowe to możliwość przyjrzenia się swoim odczuciom i zachowaniom w kontakcie z innymi ludźmi.
+        a class="btn btn-lg blue-button" href="#" role="button"
+          | więcej
+  / arrow-conrols
+  a class="left carousel-control slider-control" href="#carousel-hero" role="button" data-slide="prev"
+    i class="icon-angle-left slide-arrow" aria-hidden="true"
+    span class="sr-only"
+  a class="right carousel-control slider-control" href="#carousel-hero" role="button" data-slide="next"
```

Comment:
it can't be merged into one class (these two first are from bootstrap). Slider-control is my own class for my styles